project/Build.scala

```diff
@@ -6,63 +6,59 @@ import scalariform.formatter.preferences._
 import xerial.sbt.Sonatype._
 import xerial.sbt.Sonatype.SonatypeKeys._
 import sbtassembly.AssemblyPlugin.autoImport._
+import sbtbuildinfo.BuildInfoPlugin.autoImport._
 
 object ScalariformBuild extends Build {
 
   lazy val commonSettings = inConfig(Test)(Defaults.testSettings) ++ SbtScalariform.defaultScalariformSettings ++ sonatypeSettings ++ Seq(
     organization := "org.scalariform",
-    profileName := "org.scalariform",
+    sonatypeProfileName := organization.value,
     version := "0.2.0-SNAPSHOT",
-    scalaVersion := "2.11.7",
+    scalaVersion := crossScalaVersions.value.head,
     crossScalaVersions := Seq(
-      "2.11.7",
+      "2.11.8",
       "2.10.6"
     ),
     exportJars := true, // Needed for cli oneJar
-    scalacOptions ++= (scalaBinaryVersion.value match {
```

Comment:
The `scalacOptions` change is not valid. Scala 2.10 doesn't recognize some options.